README.md

```diff
@@ -41,3 +41,26 @@ This project is heavily inspired by [johalun/sysctl-rs](https://github.com/johal
 Kudos to @fabianfreyer for untiring support and debugging.
 
 It is developed and maintained by Stefan Grönke ([@gronke](https://github.com/gronke)) and published under [BSD 2-Clause License](LICENSE.txt).
+
+## Development
+
+### Unit Tests
+
+Unit tests may run on FreeBSD or HardenedBSD.
```

Comment:
We can add instructions for unit test execution.

```sh
python3 -m pip install pytest pytest-benchmark
pytest
```

Reply:
or, pip -r requirements-dev.txt?

Reply:
see https://github.com/gronke/py-freebsd_sysctl/pull/5 …